src/ReplayError.ts

```diff
@@ -7,6 +7,10 @@ export enum ReplayErrorType {
   ActionUndefined = 'ActionUndefined',
   /* Entity does not exist in the model */
   EntityUndefined = 'EntityUndefined',
+  /* API bad Card */
+  APIBadCard = 'APIBadCard',
+  /* API returns logicValue but has not Render */
```

Comment:
minor typo, maybe not worth to fix since you have to rerun builds:
"has no Render function" or "does not have render()"

Reply:
oh, disregard, already merged